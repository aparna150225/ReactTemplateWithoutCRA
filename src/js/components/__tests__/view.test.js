import React from 'react';
import { shallow } from 'enzyme';
import StyledP from '../view';

describe('StyledP component', () => {
  it('Renders StyledP snapshot', () => {
    const wrapper = shallow(<StyledP />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
