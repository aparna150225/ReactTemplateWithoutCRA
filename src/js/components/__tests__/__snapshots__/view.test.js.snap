// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledP component Renders StyledP snapshot 1`] = `"<StyledComponent forwardedComponent={{...}} forwardedRef={{...}} />"`;
